exotica_core/include/exotica_core/motion_solver.h

```diff
@@ -46,6 +46,7 @@ class MotionSolver : public Object, Uncopyable, public virtual InstantiableBase
     virtual void InstantiateBase(const Initializer& init);
     virtual void SpecifyProblem(PlanningProblemPtr pointer);
     virtual void Solve(Eigen::MatrixXd& solution) = 0;
+    virtual Eigen::VectorXd GetFeedbackControl(Eigen::VectorXd state, int t) const;
```

Comment:
I'd error on verge of "this doesn't belong in motion-solver". cc @VladimirIvan

Reply:
I agree.
We didn't address this in a long time but what the `Solve()` returns doesn't necessarily have to be just a joint state trajectory. For now, can you stack the whole solver output (including the state feedback) into the `solution` matrix returned by `Solve()`?

Reply:
I'd prefer not to, because it also means adding the entire reference trajectory to `solution` as well. Maybe if it returned a `struct`? Also, each solver will have different state feedback terms.

Reply:
> For now, can you stack the whole solver output (including the state feedback) into the solution matrix returned by Solve()?

That's not a great idea in my opinion. I much prefer the broken latter comment about exposing the specific solver method in Python (issue #437 related) instead. 

Alternatively, could we place it in the problem?

Reply:
So I spoke to @VladimirIvan  yesterday and he suggested we remove it from `MotionSolver` but keep it in the individual solvers; maybe have an abstract class for just that method. Or even without an abstract class. The main point is to remove it from `MotionSolver`. @wxmerkt

Reply:
In fact he said the same thing below.

Reply:
Yes, perhaps it makes sense to have `MotionSolver > FeedbackControlMotionSolver > DDP` and you can expose FeedbackControlMotionSolver (or similar) in Python.